matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/OddsType.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019 Triplebet Limited. All right reserved. Inchalla, Le Val, Alderney, GY9 3UL.
+ * Company Registration Number: 1827.
+ */
+
+package com.matchbook.sdk.core.clients.rest.dtos.prices;
+
+public enum OddsType {
+
+    DECIMAL, US
```

Comment:
i think it should be more than those ones

Reply:
Same as currencies...